pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -796,9 +796,9 @@ protected void validateGlobalNamespaceOwnership(NamespaceName namespace) {
                     validationFuture.complete(null);
                 }
             } else {
-                String msg = String.format("Policies not found for %s namespace", namespace.toString());
+                String msg = String.format("Namespace %s not found", namespace.toString());
                 log.warn(msg);
-                validationFuture.completeExceptionally(new RestException(Status.NOT_FOUND, msg));
+                validationFuture.completeExceptionally(new RestException(Status.NOT_FOUND, "Namespace not found"));
```

Comment:
Why change this section? The old comment out looks great. If the policies is non-exist, we shouldn't throw the `Namespace %s not found`.

Reply:
Not looks great. Because policies mean namespace. If there is a namespace, there are policies. For user, it's about `namespace` here. 
And some logs in Namespaces methods are also the same, not `policies` but `namespace`. Please help confirm.

Reply:
Sound good,  there is have a premise is that every namespace has policies.

Reply:
i also have the same doubt as @nodece .
the if condition on line 773 is checking if the policies exist for the namespace. this means the namespace already exists but there are no policies. it seems like the original error message is more accurate. perhaps i am missing something?

Reply:
ah ok, just saw your comment @nodece 
thanks